products/veeam-backupandreplication

```diff
@@ -0,0 +1,111 @@
+---
+title: Veeam Backup & Replication
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backupandreplication
+alternate_urls:
+-   /veeambackupandreplication
+releasePolicyLink: https://www.veeam.com/product-lifecycle.html
+activeSupportColumn: End of Fix
+releaseColumn: false
```

Comment:
https://www.veeam.com/kb2680 contains a lot of patch versions with their date, maybe we could try to follow these (we could write a script in [release-data](https://github.com/endoflife-date/release-data/) to automatically apply those patch updates).

Reply:
Sounds like a good future task.